hermeto/core/package_managers/general.py

```
@@ -99,15 +99,145 @@ async def _async_download_binary_file(
+        
+        full_image_name = f"{registry_host}/{repo}:{tag}"
+        
+        log.debug(f"Fetching OCI manifest for: {full_image_name}")
+        
+        # Use OCI Registry API to fetch manifest and download specific layer
+        registry_url = f"https://{registry_host}"
+        manifest_url = f"{registry_url}/v2/{repo}/manifests/{tag}"
+        
+        # Get Docker authentication if available
+        import json
+        import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)